src/images/badges/open-source.svg

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Please set the font color to white, and the background color to #ef4041 to work more consistently with other badges.

![image](https://user-images.githubusercontent.com/8878152/92507257-b0375b00-f1d4-11ea-837e-6b4aaafdd254.png)